src/gleam/option.gleam

```diff
@@ -323,5 +371,5 @@ pub fn lazy_or(first: Option(a), second: fn() -> Option(a)) -> Option(a) {
 /// ```
 ///
 pub fn values(options: List(Option(a))) -> List(a) {
-  list.filter_map(options, fn(op) { to_result(op, "") })
+  list_filter_map(options, fn(op) { to_result(op, "") }, [])
```

Comment:
Let's implement this directly rather than copying the filter map combinator into this module.